src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4

```diff
@@ -67,7 +67,7 @@ PREPROCESSOR: 'PREPROCESSOR';
 REPLACE: 'REPLACE';
 RETURNS: 'RETURNS';
 SHOW: 'SHOW';
-STRUCT: 'STRUCT';
+STRUCT: S T R U C T;
```

Comment:
Why this change?/

Reply:
if this is for case insensibility , it will be handled by 

https://github.com/eto-ai/rikai/blob/master/src/main/scala/ai/eto/rikai/sql/spark/parser/RikaiExtSparkSQLParser.scala#L149-L177

Reply:
make it case insensitive. Scala and Python had mismatching cases.